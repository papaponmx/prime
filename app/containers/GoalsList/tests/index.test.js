// import React from 'react';
// import { shallow } from 'enzyme';

// import { GoalsList } from '../index';

describe('<GoalsList />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
