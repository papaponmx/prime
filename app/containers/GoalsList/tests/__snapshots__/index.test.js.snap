// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GoalsList /> Should render list of goals 1`] = `
<div>
  <FormattedMessage
    defaultMessage="This is GoalsList container !"
    id="app.containers.GoalsList.header"
    values={Object {}}
  />
</div>
`;
